server/graphql/resolvers.ts

```diff
@@ -155,7 +155,7 @@ const resolvers = {
     requestTypes: (parent, __, { dataSources }): Array<RequestTypeInterface> => parent.requestTypes.map(id => dataSources.requestTypes.getById(Types.ObjectId(id)))
   },
   RequestType: {
-    numOpen: (parent, __, { dataSources }): Number => filterOpenRequests(getRequestsById(parent.requests, dataSources)).length,
+    numOpen: (parent, __, { dataSources }): number => filterOpenRequests(getRequestsById(parent.requests, dataSources)).length,
```

Comment:
Are we running the linter on existing code in this PR or another PR?

Reply:
Sure I'll run them in a separate PR.

Reply:
sweet! could we remove this diff? unless it's relevant to this PR (?)

Reply:
This diff happened because the linter will automatically fix some things on a commit.